tests/operator_test.go

```diff
@@ -38,13 +36,16 @@ var _ = Describe("Operator tests", func() {
 	f := framework.NewFramework("operator-test")
 
 	It("[test_id:3951]should create a route in OpenShift", func() {
-		if !isOpenshift(f.K8sClient) {
+		if !tests.IsOpenshift(f.K8sClient) {
 			Skip("This test is OpenShift specific")
 		}
 
 		routeClient, err := routeclient.NewForConfig(f.RestConfig)
 		Expect(err).ToNot(HaveOccurred())
 
+		routeClient, err := routeclient.NewForConfig(f.RestConfig)
+		Expect(err).ToNot(HaveOccurred())
+
```

Comment:
copy paste error?

Reply:
No unclean merge, I thought I had gotten everything, but missed that.